### DISCUSSION TITLE
整理下大家的想要的功能

### DISCUSSION THREAD

**Discussion by edocli** [edocli/emtemp-Bubble#4] [Ideas]
- [x] 首页支持关闭LOGO
- [x] 文章列表页（首页）支持随机图片
- [x] 原生外链跳转提示
- [x] 图片灯箱
- [x] 首页LOGO支持显示创始人头像
- [ ] 文章页头图支持获取文章内首张图片
- [ ] 代码折叠
- [ ] 夜间模式

欢迎大家继续提建议鸭～对于现有提议的意见也欢迎提出🥰

**Comment by Matinal111** [comment]
建议增加第三方链接跳转提示

**Reply by edocli** [reply]
好主意（*＾3＾）
